NitroxClient/GameLogic/InitialSync/VehicleInitialSyncProcessor.cs

```diff
@@ -23,11 +24,11 @@ public VehicleInitialSyncProcessor(Vehicles vehicles)
         public override IEnumerator Process(InitialPlayerSync packet, WaitScreen.ManualWaitItem waitScreenItem)
         {
             int totalSyncedVehicles = 0;
-            int nonCyclopsVehicleCount = packet.Vehicles.Where(v => v.TechType.Enum() != TechType.Cyclops).Count();
+            int nonCyclopsVehicleCount = packet.Vehicles.Count(v => v.TechType.ToUnity() != TechType.Cyclops);
 
             foreach (VehicleModel vehicle in packet.Vehicles)
             {
-                if (vehicle.TechType.Enum() != TechType.Cyclops)
+                if (vehicle.TechType.ToUnity() != TechType.Cyclops)
```

Comment:
Would it be easier to define == and != for our version of tech type?  Could we also define any implicit conversion operators for these items to make it less painful?